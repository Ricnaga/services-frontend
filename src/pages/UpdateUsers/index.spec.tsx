import { render } from '@testing-library/react';
import { UpdateUsers } from '.';

describe('UPDATE USERS SCREEN', () => {
  it('should be able to render screen', () => {
    const { container } = render(<UpdateUsers />);

    expect(container).toMatchSnapshot();
  });
});
