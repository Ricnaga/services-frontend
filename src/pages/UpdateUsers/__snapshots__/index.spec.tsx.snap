// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UPDATE USERS SCREEN should be able to render screen 1`] = `
<div>
  <div
    class="mt-4 d-flex justify-content-center container"
  >
    <div
      class="row"
    >
      <div
        class="col"
      >
        <div
          class="card"
        >
          <h1
            class="card-header"
          >
            Alterar dados do cliente
          </h1>
          <div
            class="px-4 card-body"
          >
            <form
              action="#"
              class="text-center mt-4"
            >
              <div
                class="mb-2"
              >
                <input
                  class="form-control"
                  name="id"
                  placeholder="id"
                  type="text"
                  value=""
                />
                <small
                  class="text-muted form-text"
                >
                  Preencha o 
                  id
                </small>
              </div>
              <div
                class="mb-2"
              >
                <input
                  class="form-control"
                  name="nome"
                  placeholder="nome"
                  type="text"
                  value=""
                />
                <small
                  class="text-muted form-text"
                >
                  Preencha o 
                  nome
                </small>
              </div>
              <div
                class="mb-2"
              >
                <input
                  class="form-control"
                  name="email"
                  placeholder="email"
                  type="text"
                  value=""
                />
                <small
                  class="text-muted form-text"
                >
                  Preencha o 
                  email
                </small>
              </div>
              <div
                class="mb-2"
              >
                <input
                  class="form-control"
                  name="rg"
                  placeholder="rg"
                  type="text"
                  value=""
                />
                <small
                  class="text-muted form-text"
                >
                  Preencha o 
                  rg
                </small>
              </div>
              <div
                class="mb-2"
              >
                <input
                  class="form-control"
                  name="endereco"
                  placeholder="endereco"
                  type="text"
                  value=""
                />
                <small
                  class="text-muted form-text"
                >
                  Preencha o 
                  endereco
                </small>
              </div>
              Conta
              <input
                class="form-check-input mx-2"
                name="conta"
                type="checkbox"
                value="false"
              />
              <div
                class="mt-3 d-flex justify-content-end card-footer"
              >
                <button
                  class="btn btn-outline-warning btn-lg"
                  type="submit"
                >
                  Buscar
                </button>
              </div>
            </form>
          </div>
        </div>
      </div>
    </div>
    <div
      class="toast-container position-absolute bottom-0 start-0 p-3"
    />
  </div>
</div>
`;
